src/tests/class_tests/openms/source/DataValue_test.cpp

```diff
@@ -69,49 +70,59 @@ using namespace std;
 DataValue* dv_ptr = nullptr;
 DataValue* dv_nullPointer = nullptr;
 START_SECTION((DataValue()))
-	dv_ptr = new DataValue;
+
+  // Just as a sanity check, the size of DataValue should be exactly 16 bytes
+  // on a 64 bit system:
+  // - 1 byte for the data type
+  // - 1 byte for the unit type
+  // - 4 bytes for the unit identifier (32bit integer)
+  // - 1 byte padding
```

Comment:
note that 2 bytes would actually be sufficient for the identifier for `UO` but not for the `MS` ontology which all start with a 1, e.g. `MS:1000040 ! m/z` which means we have to represent values of 1 million and above

Reply:
The 1 is just a padding number to ensure the number of digits